data/promotional-collections/highlights-japan.json

```diff
@@ -0,0 +1,227 @@
+        }
+    ],
+    "published": "2020-01-20T11:19:26Z",
+    "image": {
+        "banner": {
+            "alt": "",
+            "uri": "https://iiif.elifesciences.org/journal-cms/collection%2F2015-09%2F2d2e35c4-tropical-disease-elife-collections.png",
+            "source": {
+                "mediaType": "image/png",
+                "uri": "https://iiif.elifesciences.org/journal-cms/collection%2F2015-09%2F2d2e35c4-tropical-disease-elife-collections.png/full/full/0/default.png",
+                "filename": "tropical-disease-elife-collections.png"
+            },
+            "size": {
+                "width": 1800,
+                "height": 861
+            }
+        },
+        "thumbnail": {
+            "alt": "",
+            "uri": "https://iiif.elifesciences.org/journal-cms/collection%2F2015-09%2F2d2e35c4-tropical-disease-elife-collections.png",
+            "source": {
+                "mediaType": "image/png",
+                "uri": "https://iiif.elifesciences.org/journal-cms/collection%2F2015-09%2F2d2e35c4-tropical-disease-elife-collections.png/full/full/0/default.png",
+                "filename": "tropical-disease-elife-collections.png"
+            },
+            "size": {
+                "width": 1800,
+                "height": 861
+            }
+        }
+    },
+    "subjects": [
+        {
+            "id": "epidemiology-global-health",
+            "name": "Epidemiology and Global Health"
+        },
+        {
+            "id": "microbiology-infectious-disease",
+            "name": "Microbiology and Infectious Disease"
+        }
+    ],
+    "editors": [
+        {
+            "id": "bcooper",
+            "type": {
+                "id": "reviewing-editor",
+                "label": "Reviewing Editor"
+            },
+            "name": {
+                "preferred": "Ben Cooper",
+                "index": "Cooper, Ben"
+            }
+        },
+        {
+            "id": "pjha",
+            "type": {
+                "id": "senior-editor",
+                "label": "Senior Editor"
+            },
+            "name": {
+                "preferred": "Prabhat Jha",
+                "index": "Jha, Prabhat"
+            },
+            "image": {
+                "alt": "",
+                "uri": "https://iiif.elifesciences.org/journal-cms/person%2F2012-03%2Fkegley120208elife0213b.jpg",
+                "source": {
+                    "mediaType": "image/png",
+                    "uri": "https://iiif.elifesciences.org/journal-cms/person%2F2012-03%2Fkegley120208elife0213b.jpg/full/full/0/default.png",
+                    "filename": "kegley120208elife0213b.jpg"
+                },
+                "size": {
+                    "width": 4256,
+                    "height": 2832
+                }
+            }
+        }
+    ],
+    "content": [
+        {
+            "type": "blog-article",
+            "id": "359325",
```

Comment:
are the snippets in these two samples present in their full form in the `api-dummy` already? e.g. `/blog-articles/359325`. From the familiar titles, I think so.